src/cmd/list.go

```diff
@@ -211,12 +216,17 @@ func listOutput(images, containers []map[string]interface{}) {
 			"STATUS",
 			"IMAGE NAME")
 
+		var id, name, created, status, imageName string
 		for _, container := range containers {
-			id := container["ID"].(string)[:12]
-			name := container["Names"].(string)
-			created := container["Created"].(string)
-			status := container["Status"].(string)
-			imageName := container["Image"].(string)
+			id = container["Id"].(string)[:12]
+			if podman.CheckVersion("2.0.0") {
+				name = container["Names"].([]interface{})[0].(string)
```

Comment:
Ugh. Another JSON change. :/

I wonder if this is needed in other places.

Reply:
I have some new code in https://github.com/containers/toolbox/pull/469 that makes this a bit less painful. But yeah, we need to be careful about this.